infra/modules/services/alerts.bicep

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Consider merging Alert Action Group and Alerts into one file since they are very tightly coupled.